src/utils/csv.ts

```diff
@@ -0,0 +1,35 @@
+import { writeToString } from "@fast-csv/format";
+
+type RowInterface = {
+  [key: string]: string;
+};
+
+const getCSVRows = (rows: readonly RowInterface[]) => {
+  let results: RowInterface[] = [];
+  rows.map((row: RowInterface) => {
+    results.push(row);
+    return "";
+  });
+  return results;
+};
```

Comment:
We don't use `let` or mutate variables. There is no need to do this. The whole point of `.map` is to transform an existing thing without mutation into the new thing.

This is the cleaned equivalent of what your code here does, written in proper style:

```suggestion
const getCSVRows = (rows: readonly RowInterface[]): RowInterface[] =>
  rows.map((row: RowInterface) => row)
```

Yet my next question is what is this meant to be doing? Because it's a literal noop right now. It transforms the array of rows into the exact same array of rows.